src/app/components/publiclists/teamlist/teamlist.component.ts

```diff
@@ -119,6 +119,11 @@ export class TeamlistComponent implements OnInit, OnDestroy {
    */
   selectedTeam: any = null;
 
+  /**
+   * Currently selected team to unselect
+   */
+  otherSelectedTeam: any = null;
```

Comment:
Please change the comment and variable name as I think this is the variable holding the current clicked team. Please let me know if I'm wrong @Kajol-Kumari